app/static/js/spreadsheetMaker.js

```diff
@@ -0,0 +1,15 @@
+$(document).ready(function(){
+	$("#createSpreadsheet").on("click", function(){
+
+		$.ajax({
+			method: "POST",
+			url: "/createSpreadsheet/",
+			data: {"academicYear": $("#academicYear").val(),
+				   "academicTerm": $("#academicTerm").val()},
+			success: function(response) {
+				console.log(data)
```

Comment:
I am guessing this is a debug statement, if so it should be removed. Also, please fix the spacing for `data`

Reply:
Done